### PR TITLE
docs: add a docker install guide

### PR DESCRIPTION
**Pull request by multun**

### CODE REVIEW

File: content/docs/guides/contribute/install-docker.fr.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+Il y a deux types d'installation docker :
+- docker engine est l'application en ligne de commande
+- docker desktop est une application graphique, qui gère aussi la virtualisation
+
+Voici nos suggestions :
+- Si vous êtes sous linux, installez docker engine [via votre gestionnaire de packet](https://docs.docker.com/engine/install/#supported-platforms)
+- Si vous êtes sous MacOS / Windows, installez [docker desktop](https://www.docker.com/products/docker-desktop/) si vous y êtes authorisés
+- Si vous êtes sous windows, et voulez faire fonctionner docker sous WSL, ou ne pouvez pas utiliser docker desktop, suivez le [guide docker sous WSL](#docker-sous-wsl)
+- If you're on MacOS and can't use docker desktop, follow the [MacOS colima tutorial](#macos-colima)
```
Comment (Uriel-Sautron) [inline content/docs/guides/contribute/install-docker.fr.md:19]:
```suggestion
- Si vous êtes sous MacOS, et vous ne pouvez pas utiliser docker desktop, suivez le [guide colima pour MacOS](#macos-colima)
```

**Review by Uriel-Sautron — COMMENTED** [review]
Thank you for this guide

File: content/docs/guides/contribute/license-and-set-up.en.md [hunk 2]
```diff
@@ -28,13 +28,11 @@ Most OSRD developers use Linux (incl. [WSL](https://learn.microsoft.com/en-us/wi
 
 ### Launch the application
 
-Thanks to `docker`, one can easily compile, configure, and run all services after making a change. One can also start only a subset of the services.
```
Comment (woshilapin) [inline content/docs/guides/contribute/license-and-set-up.en.md:31]: Why did this line get removed? (and the French one is still here).

**Review by woshilapin — APPROVED** [review]
Really nice, thank you for the documentation.

File: content/docs/guides/contribute/install-docker.fr.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+Lorsqu'utilisé sous un autre système d'exploitation, docker a besoin de machines
+virtuelles linux pour build et exécuter des images.
+
+Il y a deux types d'installation docker :
+- docker engine est l'application en ligne de commande
+- docker desktop est une application graphique, qui gère aussi la virtualisation
+
+Voici nos suggestions :
+- Si vous êtes sous linux, installez docker engine [via votre gestionnaire de packet](https://docs.docker.com/engine/install/#supported-platforms)
+- Si vous êtes sous MacOS / Windows, installez [docker desktop](https://www.docker.com/products/docker-desktop/) si vous y êtes authorisés
```
Comment (bougue-pe) [inline content/docs/guides/contribute/install-docker.fr.md:17]:
```suggestion
- Si vous êtes sous MacOS / Windows, installez [docker desktop](https://www.docker.com/products/docker-desktop/) si vous y êtes autorisés
```

File: content/docs/guides/contribute/install-docker.fr.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+- Si vous êtes sous windows, et voulez faire fonctionner docker sous WSL, ou ne pouvez pas utiliser docker desktop, suivez le [guide docker sous WSL](#docker-sous-wsl)
+- If you're on MacOS and can't use docker desktop, follow the [MacOS colima tutorial](#macos-colima)
+
+
+## Docker sous WSL
+
+Cette option d'installation est très utile, car elle permet de disposer d'une installation tout à fait normale de docker engine Linux à l'intérieur de WSL, qui reste accessible depuis Windows.
+
+- [Installez WSL](https://learn.microsoft.com/fr-fr/windows/wsl/install) (Si vous avez une vielle version de WSL, lancez `wsl --upgrade`)
```
Comment (bougue-pe) [inline content/docs/guides/contribute/install-docker.fr.md:26]:
```suggestion
- [Installez WSL](https://learn.microsoft.com/fr-fr/windows/wsl/install) (Si vous avez une vieille version de WSL, lancez `wsl --upgrade`)
```

**Review by bougue-pe — APPROVED** [review]
Thanks for this, very useful for Windows onboarding 👍

File: content/docs/guides/contribute/license-and-set-up.en.md (hunk 2 above)
Comment (multun) [reply content/docs/guides/contribute/license-and-set-up.en.md:31]: I replace it with something more substancial

**Pull request merged**